Activities/Shared/UiPath.Shared.Activities/ContinuableAsyncNativeActivity.cs

```diff
@@ -16,7 +17,7 @@ protected override void Execute(NativeActivityContext context)
             }
             catch (Exception e)
             {
-                if (ContinueOnError.Get(context))
+                if (continueOnError)
```

Comment:
A better solution would be to define `ContinueOnError` as abstract in this class.

Then all derived classes would be forced to implement it properly.

See how this is done in the main Activities repository.